docs/development/releasing.rst

```diff
@@ -396,6 +416,12 @@ The procedure for this is straightforward:
 
 #. Repeat the above steps for the astropy-helpers, using the same version series.
 
+#. Create an issue titled "Release final vX.Y".
+
+#. Ping people to test it under some unusual cases like 32-bit Linux systems.
```

Comment:
does this needed? the release candidate announcement was usually picked up by the right people (debian and st packaging)

Reply:
Just following check box written by @astrofrog .

Reply:
Yeah I don't think this is needed anymore as the Debian folk follow our list.